frontend/packages/core/src/Input/select.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  FormControl as MuiFormControl,
+  InputLabel as MuiInputLabel,
+  MenuItem,
+  Select as MuiSelect,
+} from "@material-ui/core";
+import styled from "styled-components";
+
+const InputLabel = styled(MuiInputLabel)`
+  ${({ theme }) => `
+  && {
+    color: ${theme.palette.text.primary};
+  }
+  `}
+`;
+
+const FormControl = styled(MuiFormControl)`
+  ${({ theme, ...props }) => `
+  display: flex;
+  width: 100%;
+  max-width: ${props["data-max-width"] || "500px"};
+  .MuiInput-underline:after {
+    border-bottom: 2px solid ${theme.palette.accent.main};
+  }
+  `}
+`;
+
+const StyledSelect = styled(MuiSelect)`
+  ${({ ...props }) => `
+  display: flex;
+  width: 100%;
+  max-width: ${props["data-max-width"] || "500px"};
+  `}
+`;
+
+interface SelectProps {
+  defaultOption?: string;
+  label: string;
+  maxWidth?: string;
+  name: string;
+  options: string[];
```

Comment:
would it make sense to be able to specify options as as list of `labels/values`? Pretty often, we want to have a dropdown that displays one string but is backed-up by some internal value that's not visible to a user. Something like:


```typescript
interface SelectProps {
    //...
    options: Option[]
    //...
}

interface Option {
    label: string
    value: string
}
```

Reply:
Obviously, this is not a blocker - just a proposal for a small modification of API.

Reply:
💯

Reply:
Happy to help with the change - let me know and I can commit changes and push them.

Reply:
Should have it finished this morning :)